src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -303,6 +328,7 @@ private IConfidentialClientApplication GetOrBuildConfidentialClientApplication()
         /// <returns></returns>
         private IConfidentialClientApplication BuildConfidentialClientApplication()
         {
+            string instance;
             var request = CurrentHttpContext.Request;
             var microsoftIdentityOptions = _microsoftIdentityOptions;
```

Comment:
Why are these variables needed? Why not just work with the class memebers (_XXX)

Reply:
fixed throughout the class

---
In reply to: [385591146](https://github.com/AzureAD/microsoft-identity-web/pull/39#discussion_r385591146) [](ancestors = 385591146)